### PR TITLE
[4.12] OCPBUGS-6789: make the bootstrap kube-apiserver honor cluster-wide featuregates

### PR DESCRIPTION
**Pull request by stlaz**
this is a backport of https://github.com/openshift/cluster-kube-apiserver-operator/pull/1419

/cc @deads2k @ibihim

### CODE REVIEW

**Comment by deads2k** [comment]
/approve
/assign @ibihim 

for lgtm

**Review by ibihim — APPROVED** [review]
/lgtm

File: bindata/bootkube/config/bootstrap-config-overrides.yaml [hunk 1]
```diff
@@ -49,15 +49,8 @@ apiServerArguments:
...
     - {{.}}{{end}}
-  feature-gates:
-    - "APIPriorityAndFairness=true"
-    - "RotateKubeletServerCertificate=true"
-    - "DownwardAPIHugePages=true"
-    - "CSIMigrationAzureFile=false"
-    - "CSIMigrationvSphere=false"
-{{- if .ServiceCIDR | len | eq 2}}
-    - "IPv6DualStack=true"
-{{- end}}
```
Comment (ibihim) [inline bindata/bootkube/config/bootstrap-config-overrides.yaml:60]: Is there a place, where I could verify that the following flags are handed down as `.FeaturesGates`?

Comment (ibihim) [reply bindata/bootkube/config/bootstrap-config-overrides.yaml:60]: Helpful: https://github.com/openshift/cluster-kube-apiserver-operator/pull/1419/files#r1064687393

**Pull request merged**